internal/generator/generator.go

```diff
@@ -349,7 +349,9 @@ func NewGenerator(
 		buildValues.PodSecurityContext.OnRootMismatch = true
 	}
 
-	oauth2proxyDomain := CheckFeatureFlag("OAUTH2PROXY_DOMAIN", buildValues.EnvironmentVariables, generator.Debug)
+	oauth2proxyDomain := helpers.GetEnv("LAGOON_FEATURE_FLAG_OAUTH2PROXY_DOMAIN", generator.EnvironmentVariables, generator.Debug)
+	// the below doesn't work for me.
+	//oauth2proxyDomain := CheckFeatureFlag("OAUTH2PROXY_DOMAIN", buildValues.EnvironmentVariables, generator.Debug)
```

Comment:
Because [this](https://github.com/jackwrfuller/lagoon-charts/pull/1/files#diff-125718a01ae874f0e40770d01c13f787011f48ebacea675923afce3513ab73d1R254) needs to be
```
LAGOON_FEATURE_FLAG_DEFAULT_OAUTH2PROXY_DOMAIN
```
when being injected into the remote-controller